src/commands/link.ts

```diff
@@ -3,42 +3,56 @@ import path from 'path';
 import untildify from 'untildify';
 import { buildSpecFromPath } from '../';
 import BaseCommand from '../base-command';
+import BaseTable from '../base-table';
+
 declare const process: NodeJS.Process;
 
 export default class Link extends BaseCommand {
   async auth_required(): Promise<boolean> {
     return false;
   }
 
-  static description = 'Link a local component to the host to be used to power local deployments.';
+  static description = 'Link a local component to the host to be used to power local deployments or list all linked components.';
 
   static flags = {
     ...BaseCommand.flags,
   };
 
   static args = [{
     non_sensitive: true,
-    name: 'componentPath',
-    char: 'p',
+    name: 'componentPathOrList',
```

Comment:
Based on the conversation in standup, let's proceed with the ticket and do the following things:

* Create a folder in `src/commands/` called `link`
* Move `link.ts` to that new folder
* Create a new file called `index.ts` in the new folder for the linked components listing command
* Use the format `architect link:list` for the listing command

So when this is done, should have two files for the list command so that this one doesn't have to be overloaded. That should keep the code cleaner and hopefully make the usage clearer to the user since one command doesn't do two very different things